lib/extractor.rb

```diff
@@ -143,12 +143,12 @@ def get_extract(filepath)
       type_elem.value = "preview"
     end
 
-    unless @title_prefix.nil?
+    unless @title_prefix.nil? and @title_suffix.nil?
```

Comment:
D'accord avec les collègues : dès qu'il y a un and ou un or, on repasse en if en début de ligne "à l'ancienne"